src/smart-components/workspaces/workspaces.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { useIntl } from 'react-intl';
+import messages from '../../Messages';
+import { ContentHeader } from '@patternfly/react-component-groups';
+import { Card, PageSection } from '@patternfly/react-core';
+
+const Workspaces = () => {
+  const intl = useIntl();
+
+  return (
+    <React.Fragment>
+      <ContentHeader
+        title={intl.formatMessage(messages.workspaces)}
+        subtitle={intl.formatMessage(messages.workspacesSubtitle)}
+        // TODO: clarify with UX whether or not an icon will be used
+        // icon={<img src="/apps/frontend-assets/rbac-landing/rbac-landing-icon.svg" className="rbac-overview-icon" alt="RBAC landing page icon" />}
+        linkProps={{
+          label: intl.formatMessage(messages.workspacesLearnMore),
+          isExternal: true,
+          // TODO: how to pass URL to linkProps?
+          //to: 'https://access.redhat.com/documentation/en-us/red_hat_hybrid_cloud_console/2023/html/user_access_configuration_guide_for_role-based_access_control_rbac/index',
```

Comment:
```suggestion
          href: 'https://access.redhat.com/documentation/en-us/red_hat_hybrid_cloud_console/2023/html/user_access_configuration_guide_for_role-based_access_control_rbac/index',
```
linkProps extend PF ButtonProps, so `href` can be used

Reply:
Fixed, thank you